content/support-matrix/delfin/dell-emc/vmax/_index.md

```diff
@@ -0,0 +1,363 @@
+      <tr>
+         <td rowspan=18>Device</td>
+         <td>Name</td>
+         <td>Device Name</td>
+      </tr>
+      <tr>
+         <td>Vendor</td>
+         <td>Vendor</td>
+      </tr>
+      <tr>
+         <td>Description</td>
+         <td>Device Description</td>
+      </tr>
+      <tr>
+         <td>Model</td>
+         <td>Device Model</td>
+      </tr>
+      <tr>
+         <td>Firmware version</td>
+         <td>Firmware version</td>
+      </tr>
+      <tr>
+         <td>Status</td>
+         <td>It shows availability of device ex: offline, normal etc.</td>
+      </tr>
+      <tr>
+         <td>Serial Number</td>
+         <td>It is VMAX array id</td>
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td>Device Location, is name-value pair having component type and component name</td>
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>It is measured in terms of Tera byte</td>
+      </tr>
+      <tr>
+         <td>Used Capacity</td>
+         <td>It is measured in terms of Tera byte</td>
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>It is measured in terms of Tera byte</td>
+      </tr>
+      <tr>
+         <td>Raw Capacity</td>
+         <td>It is measured in terms of Giga byte</td>
+      </tr>
+      <tr>
+         <td>Subscribed Capacity</td>
+         <td>Refers to capacity allocated to device/host</td>
+      </tr>
+   </tbody>
+</table>
+ 
+### Alert data matrix
+ 
+<table>
+   <thead>
+      <tr>
+         <th>Attribute</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Alert id</td>
+         <td>Identification of alarm.</td>
+      </tr>
+      <tr>
+         <td>Alert name</td>
+         <td>Name of the alarm</td>
+      </tr>
+      <tr>
+         <td>Severity</td>
+         <td>Severity of the alarm</td>
+      </tr>
+      <tr>
+         <td>Category</td>
+         <td>Type of the reported alarm notification. Ex: New, Fixed, Removed etc.</td>
+      </tr>
+      <tr>
+         <td>Type</td>
+         <td>Basic classification of the alarm. Ex:CommunicationsAlarm, EquipmentAlarm</td>
+      </tr>
+      <tr>
+         <td>Sequence number</td>
+         <td>It maps to the trap</td>
```

Comment:
Unique identification for the alert instance

Reply:
done